.github/workflows/ci-cd-pipeline-main.yml

```
@@ -0,0 +1,36 @@
+# This workflow will run whenever commits land on the main branch.
+#
+# It will run tests and a few safety checks.
+#
+# If everything passes it can be set to auto-deploy to your production app on Heroku.
+#
+# This workflow is primarily meant to be triggered automatically, but it can be run manually.
+name: " 🎥 Main CI/CD Pipeline (prod)"
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  mini_test:
+    name: 🧪 MiniTest
```

Minor nit but technically the name is [minitest](https://github.com/minitest/minitest/), don't know if we want to lowercase the m though 🙂

```suggestion
  minitest:
    name: 🧪 Minitest
```